streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -87,25 +97,171 @@ public static void main(String[] args) throws Exception {
         System.out.println("kafka=" + kafka);
         System.out.println("zookeeper=" + zookeeper);
         System.out.println("stateDir=" + stateDir);
+        System.out.println("numRecords=" + numRecords);
 
         SimpleBenchmark benchmark = new SimpleBenchmark(stateDir, kafka, zookeeper);
 
         // producer performance
-        benchmark.produce();
+        benchmark.produce(SOURCE_TOPIC, VALUE_SIZE, "simple-benchmark-produce", numRecords, true, numRecords, true);
         // consumer performance
-        benchmark.consume();
+        benchmark.consume(SOURCE_TOPIC);
         // simple stream performance source->process
-        benchmark.processStream();
+        benchmark.processStream(SOURCE_TOPIC);
         // simple stream performance source->sink
-        benchmark.processStreamWithSink();
+        benchmark.processStreamWithSink(SOURCE_TOPIC);
         // simple stream performance source->store
-        benchmark.processStreamWithStateStore();
+        benchmark.processStreamWithStateStore(SOURCE_TOPIC);
+        // simple streams performance KSTREAM-KTABLE join
+        benchmark.kStreamKTableJoin(JOIN_TOPIC_1_PREFIX + "kStreamKTable", JOIN_TOPIC_2_PREFIX + "kStreamKTable");
+        // simple streams performance KSTREAM-KSTREAM join
+        benchmark.kStreamKStreamJoin(JOIN_TOPIC_1_PREFIX + "kStreamKStream", JOIN_TOPIC_2_PREFIX + "kStreamKStream");
+        // simple streams performance KTABLE-KTABLE join
+        benchmark.kTableKTableJoin(JOIN_TOPIC_1_PREFIX + "kTableKTable", JOIN_TOPIC_2_PREFIX + "kTableKTable");
```

Comment:
It's less about multiple classes and more about being able to track how the performance of the various operations varies over time. Would it be useful to know that joins became slower instead of `processStreamWithStateStore`? If so, then it would make sense to have separate benchmarks. If not, the current way is fine.